CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 ## 3.27.0 (Unreleased)
 
+IMPROVEMENTS:
+* accesscontextmanager: Added `custom` config to `google_access_context_manager_access_level` ([#2180](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2180))
+* cloudbuild: Added invert_regex flag in Github PullRequestFilter and PushFilter + in triggerTemplate ([#2171](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2171))
```

Comment:
```suggestion
* cloudbuild: Added `invert_regex` flag in Github PullRequestFilter and PushFilter + in triggerTemplate ([#2171](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2171))
```

Reply:
What is the "+" about?

Reply:
🤷 it was in the original message, not sure